src/autoPagination.ts

```diff
@@ -362,7 +351,7 @@ function wrapAsyncIteratorWithCallback<T>(
         onItem(item, next);
       }).then((shouldContinue) => {
         if (shouldContinue === false) {
-          return handleIteration({done: true});
+          return handleIteration({done: true, value: undefined});
```

Comment:
This is the only real code change. It's not breaking because in `handleIteration`, the argument is only used like
```
      const item = iterResult.value;
      return new Promise((next) => {
        onItem(item, next);
```
and iterResult.value will be 'undefined' regardless of whether .value is absent or .value is explicitly undefined.